selfapi.js

```diff
@@ -367,6 +372,21 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  toAPIIndex: function (basePath) {
```

Comment:
Nit: If you're intending to pass URLs to this function (e.g. `http://localhost:8080`) instead of paths (e.g. `/api`), please call this argument `baseUrl`.